Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -964,4 +1049,135 @@ Common::Matrix44 Wiimote::GetTotalTransformation() const
       Common::Quaternion::RotateX(m_imu_cursor_state.recentered_pitch)));
 }
 
+std::string BalanceBoard::GetName() const
+{
+  return "BalanceBoard";
+}
+
+void BalanceBoard::LoadDefaults(const ControllerInterface& ciface)
+{
+  EmulatedController::LoadDefaults(ciface);
+
+  // Power
+  m_buttons->SetControlExpression(0, "P");
+
+  // Balance: Up, Down, Left, Right
+  m_balance->SetControlExpression(0, "I");
+  m_balance->SetControlExpression(1, "K");
+  m_balance->SetControlExpression(2, "J");
+  m_balance->SetControlExpression(3, "L");
+
+  // Step Off
+  m_options->SetControlExpression(0, "O");
+
+  // Because our defaults use keyboard input, set calibration shape to a square.
+  m_balance->SetCalibrationFromGate(ControllerEmu::SquareStickGate(1.0));
```

Comment:
In the previous version, I used `ControllerEmu::SquareStickGate(.5)`. I don't entirely remember why, but it might have been something to do with leaning too far causing problems or being very hard to control with a keyboard (and also acting more like standing on the balance board with 1 foot instead of leaning). But it's been a while so I don't remember the exact reasons.

Reply:
I'm guessing you were having weirdness combined with the way sensor values were calculated. The corners opposite of the center-of-balance were getting negative values. That's resolved now.

Currently, full left/right COB is the equivalent of standing on the extreme edge of the board, which is maybe a bit odd. I'm considering adding a "Foot Separation" setting to control that.